docs/sources/datasources/prometheus/_index.md

```diff
@@ -14,14 +14,22 @@ weight: 1300
 
 # Prometheus data source
 
-Grafana ships with built-in support for Prometheus.
-This topic explains options, variables, querying, and other features specific to the Prometheus data source, which include its [feature-rich code editor for queries and visual query builder]({{< relref "./query-editor/" >}}).
+Prometheus is an open-source toolkit that collects and stores metrics used for monitoring and alerting. If you are just getting started with Prometheus, see [What is Prometheus?](https://grafana.com/docs/grafana/next/fundamentals/intro-to-prometheus/)
```

Comment:
This should be a relref.

Reply:
I am inclined to leave as is since we are looking at changing how links are handled...is that fair?

Reply:
changed to rel ref